ibm_catalog.json

```
@@ -0,0 +1,103 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-icd-redis",
+      "label": "ICD Redis Module",
+      "product_kind": "module",
+      "tags": [
+        "dev_ops",
+        "target_terraform",
+        "terraform",
+        "module"
+      ],
+      "keywords": [
+        "terraform",
+        "redis",
+        "icd"
+      ],
+      "short_description": "Implements an instance of IBM Cloud Databases for Redis.",
+      "long_description": "Use this module to provision and configure an IBM Cloud Database for Redis instance.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-icd-redis/main/images/redis_icon.svg",
+      "features": [
+        {
+          "title": "Create Redis instance",
+          "description": "Create IBM Cloud Database for Redis instance."
+        },
+        {
+          "title": "Attach Users",
+          "description": "Attach list of users to the Redis instance."
+        },
+        {
+          "title": "Auto Scaling instance",
+          "description": "Rules to allow the Redis database instance to increase resources in response to usage."
+        },
+        {
+          "title": "Create Service Credentials Roles",
+          "description": "Create Service Credentials Roles for the ICD Redis database instance."
+        },
+        {
+          "title": "KMS encryption",
+          "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
+        },
```

```suggestion
          "title": "Auto scaling instance",
          "description": "Supports rules that allow the Cloud Databases instance to increase resources in response to usage."
        },
        {
          "title": "Create service credentials roles",
          "description": "Create service credentials roles for the instance of Databases for Redis."
        },
        {
          "title": "KMS encryption",
          "description": "Supports key management encryption (BYOK and KYOK), including service-to-service authorization policies with KMS."
        },
```

What are service credential roles? Are they really service credentials or can you create roles with different permissions?

---

A service credential can be assigned an IAM role such as `Administrator`, `Operator`, `Viewer` or `Editor`